Add runtime.on performance warning (#32222)
mdn/content: files/en-us/mozilla/add-ons/webextensions/api/runtime/onperformancewarningcategory/index.md

---
title: runtime.OnPerformanceWarningCategory
slug: Mozilla/Add-ons/WebExtensions/API/runtime/OnPerformanceWarningCategory
page-type: webextension-api-type
browser-compat: webextensions.api.runtime.OnPerformanceWarningCategory
---

{{AddonSidebar}}

The category of warning that dispatched the {{WebExtAPIRef("runtime.onPerformanceWarning")}} event.

## Type

Values of this type are strings. Possible values are:

> - `"content_script"`: The performance warning originated from a content script.

Reviewer: ```suggestion
- `"content_script"`: The performance warning relates to a slow content script.
```

> - `"content_script"`: The performance warning originated from a content script.

Author: Revised following feedback from @Rob--W 
```suggestion
- `"content_script"`: The performance warning is for a slow content script in this extension.
```
